rgw/admin/quota.go

```diff
@@ -0,0 +1,56 @@
+package admin
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+)
+
+// Quota describes an object store quota for a user or a bucket
+type Quota struct {
+	UID        string `json:"user_id" url:"uid"`
+	QuotaType  string `url:"quota-type"`
+	Enabled    *bool  `json:"enabled" url:"quota,enabled"`
+	CheckOnRaw bool   `json:"check_on_raw"`
+	MaxSize    *int64 `json:"max_size" url:"max-size"`
```

Comment:
This is bit confusing for me. Here the type is `int` pointer, so what unit for size Bytes, KB, MB.
From the usability point of view, IMO it better pass this as string. So that the user can specify 10KB, 10MB and 10GB etc

Reply:
And then do some internal conversion? Even though it brings benefits to the user, it's easy to get it wrong like someone passing `GB`, or `Gb` or `GiB` or `gib` etc...

Reply:
@leseb : What about using enums for GB,KB,TB etc and keep `MaxSize` as int. The `radosgwadmin` command accepts size unit while setting quota AFAIR. Can you please try whether `string` works for AdminOps api.

Reply:
Testing now

Reply:
Oh I see.